remo/base/views.py

```diff
@@ -38,12 +39,22 @@ def login_failure(self, msg=''):
         return super(OIDCCallbackView, self).login_failure()
 
 
-@cache_control(private=True, no_cache=True)
+@cache_control(private=True, max_age=60 * 60 * 8)
 def main(request):
     """Main page of the website."""
     featured_rep = utils.latest_object_or_none(FeaturedRep)
-    feed = feedparser.parse(settings.PLANET_URL)
-    planet_entries = feed.entries[:3]
+
+    try:
+        resp = requests.get(settings.PLANET_URL, timeout=settings.PLANET_MAX_TIMEOUT)
+    except requests.exceptions.ConnectionError:
+        resp = ''
+        planet_entries = ''
```

Comment:
I simplified the code a bit, but this doesn't work for me. I mean the unicode part. I tried this with a non-unicode rss feed and it don't work properly.